crates/primitives/src/eip4844.rs

```diff
@@ -1,6 +1,15 @@
 //! Helpers for working with EIP-4844 blob fee
+use crate::{kzg::KzgCommitment, H256, constants::eip4844::{TARGET_DATA_GAS_PER_BLOCK, VERSIONED_HASH_VERSION_KZG}};
 
-use crate::constants::eip4844::TARGET_DATA_GAS_PER_BLOCK;
+/// Calculates the versioned hash for a KzgCommitment
+///
+/// Specified in [EIP-4844](https://eips.ethereum.org/EIPS/eip-4844#header-extension)
+pub fn kzg_to_versioned_hash(commitment: KzgCommitment) -> H256 {
+    use sha2::{Sha256, Digest};
```

Comment:
style nit: would just like this import at the top of the file rather than within the fn